### PR TITLE
UI component Grid Loading logger issue

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
- **Ticket** : [JDTB-26](https://wearejh.atlassian.net/browse/JDTB-26)
- Fixed the Issue of JH Logger Module UI Component Grid Issue.
- Fixed timestamp issue

### CODE REVIEW

**Review by bartoszkubicki — APPROVED** [review]

File: Service/LoggerManagement.php [hunk 1]
```diff
@@ -84,6 +84,7 @@ public function addEvent(
         $log->setSeverity($severity);
         $log->setIdentifierLabel($identifierLabel);
         $log->setIdentifierValue($identifierValue);
+        $log->setTimestamp(strtotime(time()));
```
Comment (pknap) [inline Service/LoggerManagement.php:87]: @RakeshJesadiya

image

How was that supposed to work?

@bartoszkubicki

Comment (RakeshJesadiya) [reply Service/LoggerManagement.php:87]: @pknap It's might not have worked but while I was testing on woodies it was working and added a time log to the database table.
I don't see any error while testing and still it is working but it's syntactically wrong so I will update the PR.

Comment (pknap) [reply Service/LoggerManagement.php:87]: @RakeshJesadiya,

Could you please elaborate what do you mean by syntactically wrong? It is just wrong

https://www.php.net/manual/en/function.strtotime.php

strtotime — Parse about any English **textual** **datetime** **description** **into** a **Unix** **timestamp**

https://www.php.net/manual/en/function.time.php

time — **Return** **current** **Unix** **timestamp**

You are passing timestamp into a function that should convert textual date into timestamp. How on earth could this have worked?

Comment (RakeshJesadiya) [reply Service/LoggerManagement.php:87]: @pknap Yes, It's (strtotime(time())) still working on PHP. that's the magic of PHP or whatever not sure. **But that must need to be thrown an error but it was working locally while I tested it that's not correct.**

**strtotime(time())** while run time needs to throw an error but that will bypass it in woodies local and add the current time in the DB field.

**Pull request merged**